app/src/main/java/com/example/mytestproject/util/MySharedPreferences.kt

```diff
@@ -0,0 +1,18 @@
+package com.example.mytestproject.util
+
+import android.content.Context
+import android.util.Log
+import com.example.domain.models.CityModel
+import com.google.gson.Gson
+import javax.inject.Inject
+
+class MySharedPreferences(private val context: Context){
```

Comment:
SharedPreferences could be injected here instead of context.

Reply:
Fixed